internal/observability.go

```diff
@@ -30,12 +31,13 @@ import (
 	"go.opencensus.io/trace"
 
 	commonpb "github.com/census-instrumentation/opencensus-proto/gen-go/agent/common/v1"
-	tracepb "github.com/census-instrumentation/opencensus-proto/gen-go/trace/v1"
 )
 
 var (
 	tagKeyReceiverName, _ = tag.NewKey("opencensus_receiver")
 	tagKeyExporterName, _ = tag.NewKey("opencensus_exporter")
+	tagKeyError, _        = tag.NewKey("opencensus_error")
```

Comment:
The prefix "opencensus_" seems a bit to verbose to me: the namespace already should give that this is OpenCensus and then the paris will look strange: opencensus_receiver="zipkin". The case that makes sense is if for instance status is being mapped to the OC status defined in proto.